tests/foreman/longrun/test_oscap.py

```diff
@@ -230,6 +233,127 @@ def test_positive_oscap_run_via_ansible(
         assert result is not None
 
 
+@pytest.fixture
+def scap_prerequisites(module_org, default_proxy, target_sat):
+    # TODO: add support for RHEL9 (it doesn't have scap content in Sat by default) and parametrize distro
+    distro = 'rhel8'
+    profile = OSCAP_PROFILE['ospp8']
+    content = OSCAP_DEFAULT_CONTENT[f'{distro}_content']
+    hgrp_name = gen_string('alpha')
+    policy_name = gen_string('alpha')
+    # Create hostgroup
+    target_sat.cli_factory.hostgroup(
+        {
+            'content-source-id': default_proxy,
+            'name': hgrp_name,
+            'organizations': module_org.name,
+        }
+    )
+    # Create oscap policy
+    scap_id, scap_profile_id = fetch_scap_and_profile_id(target_sat, content, profile)
+    target_sat.cli.Ansible.roles_import({'proxy-id': default_proxy})
+    target_sat.cli.Ansible.variables_import({'proxy-id': default_proxy})
+    role_id = target_sat.cli.Ansible.roles_list({'search': 'foreman_scap_client'})[0].get('id')
+    target_sat.cli_factory.make_scap_policy(
+        {
+            'scap-content-id': scap_id,
+            'hostgroups': hgrp_name,
+            'deploy-by': 'ansible',
+            'name': policy_name,
+            'period': OSCAP_PERIOD['weekly'].lower(),
+            'scap-content-profile-id': scap_profile_id,
+            'weekday': OSCAP_WEEKDAY['friday'].lower(),
+            'organizations': module_org.name,
+        }
+    )
+    return hgrp_name, role_id, distro
+
+
+@pytest.mark.e2e
+@pytest.mark.tier4
+def test_positive_oscap_remediation(
+    module_org, default_proxy, content_view, lifecycle_env, target_sat, scap_prerequisites
+):
+    """Run an OSCAP scan and remediate through WebUI
+
+    :id: 55b919ef-432f-4186-b22a-01bb8ce39b3f
+
```

Comment:
This token can be added
```suggestion
:Verifies: SAT-23240
```